docs_site/docusaurus.config.js

```diff
@@ -151,6 +151,12 @@ const config = {
       },
     }),
 
+  scripts: [
```

Comment:
future-work: These scripts should not be here, since they only affect a small subset of pages.